parking_permits/talpa/order.py

```diff
@@ -157,10 +158,10 @@ def _create_order_data(cls, order):
                     order_items_of_single_permit.append(item)
 
             # Append details of permit only to the last order item of permit.
-            cls._append_detail_meta(order_items_of_single_permit[-1], permit)
+            cls.append_detail_meta(order_items_of_single_permit[-1], permit)
```

Comment:
What happens if `order_items_of_single_permit` is empty? Should probably be a length check or `IndexError `handling here.